atomate/vasp/workflows/base/surface.py

```diff
@@ -0,0 +1,160 @@
+# coding: utf-8
+
+
+## for Surface Energy Calculation
+from __future__ import division, unicode_literals
+
+"""
+If you use this module, please consider citing the following work::
+
+    R. Tran, Z. Xu, B. Radhakrishnan, D. Winston, W. Sun, K. A. Persson,
+    S. P. Ong, "Surface Energies of Elemental Crystals", Scientific Data,
+    2016, 3:160080, doi: 10.1038/sdata.2016.80.
+
+as well as::
+
+    Sun, W.; Ceder, G. Efficient creation and convergence of surface slabs,
+    Surface Science, 2013, 617, 53–59, doi:10.1016/j.susc.2013.05.016.
+"""
+
+__author__ = "Richard Tran"
+__version__ = "0.2"
+__email__ = "rit001@eng.ucsd.edu"
+__date__ = "11/30/17"
+
+import os
+
+from fireworks import Workflow
+
+from atomate.vasp.fireworks.core import SurfCalcOptimizer
+
+
+class SurfaceWorkflowManager(object):
```

Comment:
I am not sure the SurfaceWorkflowManager is the best name - it's not really managing workflows, it's more responsible to help create them? Maybe SurfaceWorkflowCreator?